test/parallel/test-http2-server-set-invalid-header.js

```diff
@@ -0,0 +1,56 @@
+// Flags: --expose-http2
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const http2 = require('http2');
+const body =
+  'Cannot set HTTP/2 pseudo-headers';
+
+const invalidHeaders = [
+  { key: ':status', value: 200 }, 
+  { key: ':method', value: 'GET' }, 
+  { key: ':path', value: '/' }, 
+  { key: ':authority', value: 'example.com' }, 
+  { key: ':scheme', value: 'http' }
+];
+
+const checkServer = (header, value) => {
+  const server = http2.createServer((req, res) => {
+    try {
+      res.setHeader('foobar', 'baz');
```

Comment:
Move these first two `setHeader` calls out of the `try` block. They should never throw.